machine_learning/benchmarks/circuit_qnn_classifier_benchmark.py

```diff
@@ -73,7 +70,59 @@ def parity(x):
         self.circuit_classifier_fitted = NeuralNetworkClassifier(
             neural_network=self.circuit_qnn, optimizer=COBYLA()
         )
-        self.circuit_classifier_fitted.fit(self.X, self.y01)
+        self.circuit_classifier_fitted.fit(X, y)
+
+    def setup_dataset_iris(self, X, y, quantum_instance_name):
+        """Training CircuitQNN function for iris dataset."""
+
+        self.output_shape = 3
+
+        # scaling data
+        scaler = MinMaxScaler((-1, 1))
+        self.X = scaler.fit_transform(X)
+
+        # creating feature map
+        feature_dim = self.X.shape[1]
+        feature_map = ZFeatureMap(feature_dim)
```

Comment:
Why is it `ZFeatureMap` while `ZZFeatureMap` in the synthetic case? Is the score or training time better?
In general, setup should be identical in the similar tests, easier to compare.

Reply:
Updated so both synthetic and iris dataset models use ZZFeatureMap